fe/fe-core/src/main/java/com/starrocks/sql/InsertPlanner.java

```diff
@@ -222,10 +222,30 @@ public ExecPlan plan(InsertStmt insertStmt, ConnectContext session) {
                     Preconditions.checkState(!CollectionUtils.isEmpty(targetPartitionIds));
                     enableAutomaticPartition = olapTable.supportedAutomaticPartition();
                 }
+                boolean nullExprInAutoIncrement = false;
+                // In INSERT INTO SELECT, if AUTO_INCREMENT column
+                // is specified, the column values must not be NULL
+                if (!(queryRelation instanceof ValuesRelation)) {
```

Comment:
Does the CTAS or materialized view also need to modify?

Reply:
CTAS need it and materialized view maybe not, I have update the code